ui/app/styles/components.scss

```diff
@@ -1,12 +1,12 @@
 @import './components/accordion';
-@import './components/badge';
+@import './components/badge-nomad-internal';
```

Comment:
@Dhaulagiri Is this a sufficient way to get around naming collision we chatted about in Slack? I clicked around our application and everything appears to be just fine.

Reply:
yes, as long as it isn't exactly `components/badge` you should avoid the conflict

Reply:
Awesome! cc:  @philrenaud